packages/wallets/src/features/cashier/modules/Transfer/components/TransferMessages/TransferMessages.scss

```diff
@@ -7,6 +7,8 @@
     }
 
     &__link {
+        display: flex;
+        width: max-content;
```

Comment:
To fix the following UI issue on certain languages:
<img width="789" alt="Bildschirmfoto 2024-11-27 um 2 29 27 PM" src="https://github.com/user-attachments/assets/248dd8ce-ad71-4cc3-8169-be4cb4fe1ceb">